style.css

```diff
@@ -1,395 +1,635 @@
+  padding: 10px 0 10px 10px;
+}
+
+/* Contact form */
+
+.contact {
+  width: 370px;
+  height: 779px;
+  background: #ffff;
+}
+
+.contact-form {
+  background: #6070ff;
+  border: none;
+  width: 370px;
+  height: 789px;
+  margin-left: -1px;
+  border-top-left-radius: 80px;
+  padding: 0;
+}
+
+.background-img {
+  background-image: url("images/contact-form-mobile.svg");
+  height: 709px;
+  background-repeat: no-repeat;
+  background-position: right;
+  margin-top: 30px;
+}
+
+.contact-heading {
+  color: #fff;
+  text-align: center;
+  font-family: "Poppins", Arial, Helvetica, sans-serif;
+  font-size: 40px;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 52px;
+  position: absolute;
+  margin-top: 80px;
+  width: 345px;
+}
+
+.contact-details {
+  color: #ebebff;
+  text-align: center;
+  font-family: "Poppins", Arial, Helvetica, sans-serif;
+  font-size: 20px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 28px;
+  margin-top: 140px;
+  width: 325px;
+  margin-left: 20px;
+  position: absolute;
+}
+
+form {
+  position: absolute;
+  margin-top: 280px;
+  padding: 20px;
+  width: 375px;
+}
+
+#username,
+#email {
+  width: 327px;
+  height: 48px;
+  border-radius: 8px;
+  border: 1px solid #cfd8dc;
+  background: #fff;
+  margin-bottom: 20px;
+  color: #172b4d;
+  font-family: "Poppins", Arial, Helvetica, sans-serif;
+  font-size: 17px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 20px;
+  padding-left: 5px;
+}
+
+#textarea {
+  width: 327px;
+  height: 170px;
+  border-radius: 8px;
+  border: 1px solid #cfd8dc;
+  background: #fff;
+  color: #b3bac5;
+  font-family: "Poppins", Arial, Helvetica, sans-serif;
+  font-size: 15px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 24px;
+  padding: 5px;
+  margin-bottom: 20px;
+}
+
+.contact-btn {
+  border-radius: 8px;
+  background: #fff;
+  padding: 12px 16px;
+  color: #6070ff;
+  text-align: center;
+  font-family: "Poppins", Arial, Helvetica, sans-serif;
+  font-size: 17px;
+  font-style: normal;
+  font-weight: 500;
+  line-height: 24px;
+  letter-spacing: 0.51px;
+  border: 1px solid #fff;
+}
+
+.footer-div {
+  width: 134px;
+  height: 5px;
+  border-radius: 10px;
+  background: #0d151e;
+  margin: -10px 120px;
+}
+
+.contact-btn:hover,
+.see:hover,
+.abt:hover {
+  cursor: pointer;
+  background-color: #6070ff;
+  color: white;
+  transform: scale(1.08);
+}
+
+.a:hover {
+  cursor: pointer;
+}
+
+.fa-brands:hover {
+  color: #6070ff;
+  transform: scale(1.08);
+}
+
+.contact-btn:active,
+.see:active,
+.abt:active {
+  cursor: pointer;
+  background-color: darkblue;
+  color: white;
+  transform: scale(1.08);
+}
+
+.a:active {
+  cursor: pointer;
+}
+
+.fa-brands:active {
+  color: #6070ff;
+  transform: scale(1.08);
+}
+
+@media screen and (max-width: 768px) {
+  .menu,
+  .cardImgdesk,
+  .tonic1,
+  .Dev-desk,
+  .counter-desk,
+  .canopyHeadingDesk {
+    display: none;
+  }
+}
+
+@media only screen and (min-width: 768px) {

```

Comment:
At exactly `min-width: 768px` it is expected that all the elements in your app displayed properly without any horizontal overflow(horizontal overflow does not give good user experience). The screenshot below clarifies it further. ℹ️ 

<img width="589" alt="Untitled" src="https://github.com/garikaipaidamoyo/Portfolio/assets/103386183/63a080ea-8465-4fb7-8d13-4f6225a7f78c">